src/main/kotlin/com/getkeepsafe/dexcount/Plugin.kt

```diff
@@ -101,17 +101,17 @@ abstract class TaskProvider(
         val variants: DomainObjectCollection<out BaseVariant> = when {
             project.plugins.hasPlugin("com.android.application") -> {
                 val ext = project.extensions.findByType(AppExtension::class.java)
-                ext.applicationVariants
+                ext!!.applicationVariants
```

Comment:
Updating to Gradle 4.2 caused the build to fail thinking this could be nullable.

Reply:
Can you use `?.`?